### DISCUSSION TITLE
Qt GUI might be better.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [hardinfo2/hardinfo2#53] [Ideas]
For the basic, functional, and intuitive interface that HardInfo is known for, I'm uncertain that GTK3+ (especially GTK4+) are suitable. A simple `QMainWindow` with a `TreeView` and right-hand pane might actually yield better results with less custom code.

**Comment by hwspeedy** [comment]
Any improvement suggestions are welcome - make a PR showing the features we gain.

So far I have no issues with GTK3+ - and the natural roadmap is GTK4+ (some ~20 years from now)

**Reply by roke-julian-lockhart** [reply]
> https://github.com/hardinfo2/hardinfo2/discussions/53#discussioncomment-10052905

@hwspeedy, if your timeline for GTK4 is 20 years, then we might not need to worry, since GTK3 is pretty acceptable for most situtions. I'll make an example when I can, though it might take a year or two for me to have the time (not that that seems like a problem for ya, luckily).
